CMakeLists.txt

```diff
@@ -402,13 +425,15 @@ if(DESKTOP)
         ${UWEBSOCKETS_SOURCE_DIR}/src/WebSocket.cpp)
     if(MSVC)
       set(websockets_additional_defines
+        ${websockets_additional_defines}
```

Comment:
you might be able to use [list append](https://cmake.org/cmake/help/v3.12/command/list.html#append) here.  Just a thought.

Reply:
Hmm, do we tend to prefer one or the other in our cmake files? I think I see more instances of set(${thing} ${thing} additional items) rather than list append

Reply:
Meh.  Thought it might be easier to read.  I don't have a strong opinion about it, though.